### DISCUSSION TITLE
我是治安官，请协助我调查一个人

### DISCUSSION THREAD

**Discussion by WRYYY-S** [share121/inter-knot#613] [其他]
最近我在追查一个人，我需要更多这个人的照片来确定她的身份，上头不给人手，只能请求神通广大的绳网用户了！
![Hoshimi Miyabi](https://www.z4a.net/images/2024/08/10/Hoshimi-Miyabi.md.jpg)
![hoshimi miyabi batouzan](https://www.z4a.net/images/2024/08/10/hoshimi-miyabi-batouzan.md.jpg)
![hoshimi miyabi pull out katana](https://www.z4a.net/images/2024/08/10/hoshimi-miyabi-pull-out-katana.md.jpg)
![katana water watermark weapon hoshimi](https://www.z4a.net/images/2024/08/10/katanawater-watermark-weapon-hoshimi.md.jpg)

**Comment by Bilistar333** [comment]
好图

**Comment by Stewiegriffin6** [comment]
这个委托我接了

**Comment by Tragedysaver1024** [comment]
南无三，面对被治安局渗透的绳网，就连佛陀也不禁流下血泪……

**Comment by Jakobboy** [comment]
帅

**Comment by 2132925099** [comment]
.--.  ..  -.-.  -  ..-  .-.  .  ...  .-  ...-  .  -..

**Comment by Micawber-jock** [comment]
收手吧治安官，外面全是对空部

**Comment by zifeiyu1111** [comment]
想骗图就直说，发我一份
